app/controllers/v0/claims_controller.rb

```diff
@@ -4,13 +4,18 @@ class ClaimsController < ApplicationController
     skip_before_action :authenticate
 
     def index
-      render json: current_user.claims,
+      render json: Claim.fetch_all(current_user.vaafi_headers),
```

Comment:
I agree that the evss clients in /lib should not instantiate with model objects, but I think it would be okay for claim to access a user's vaafi headers instead of the controller doing it.